tuxemon/log.py

```diff
@@ -46,34 +130,123 @@ def configure():
     }
     config = prepare.CONFIG
     loggers = {}
-
     if config.debug_level in LOG_LEVELS:
         log_level = LOG_LEVELS[config.debug_level]
+
     else:
         log_level = logging.INFO
-
     # Set up logging if the configuration has it enabled
     if config.debug_logging:
 
         for logger_name in config.loggers:
 
             # Enable logging for all modules if specified.
             if logger_name == "all":
+
                 print("Enabling logging of all modules.")
                 logger = logging.getLogger()
             else:
                 print("Enabling logging for module: %s" % logger_name)
                 logger = logging.getLogger(logger_name)
-
             # Enable logging
             logger.setLevel(log_level)
+
             log_hdlr = logging.StreamHandler(sys.stdout)
             log_hdlr.setLevel(log_level)
             log_hdlr.setFormatter(logging.Formatter("%(asctime)s - %(name)s - " "%(levelname)s - %(message)s"))
             logger.addHandler(log_hdlr)
+            # Archive previous log
+            archive_log()
 
+            # Logging to file
+            log_filehandler = logging.FileHandler(f"{USER_LOG_DIR}/latest.log")
+            log_filehandler.setLevel(logging.INFO)
+            log_filehandler.setFormatter(logging.Formatter("%(asctime)s - %(name)s - " "%(levelname)s - %(message)s"))
+            logger.addHandler(log_filehandler)
             loggers[logger_name] = logger
 
+
             # prevent pyscroll redraw warnings
             pyscroll_logger = logging.getLogger("orthographic")
             pyscroll_logger.setLevel(logging.ERROR)
+
+def send_logs():
+    config = prepare.CONFIG
+    logging.info("="* 26)
+
+    logging.info("=== System information ===")
+
+    # OS Info
+    entries = {
+        platform.platform: "platform",
+        platform.processor: "processor",
+        platform.python_version: "python_version",
+        platform.python_implementation: "python_implementation",
+    }
+    for i in entries:
+        logging.info(f"{entries[i]}\t{i()}")
+
+    logging.info("=== Library versions ===")
+
+    lib_entries = {
+        "babel": "babel",
+        "cbor": "cbor",
+        "neteria": "neteria",
+        "pillow": "PIL",
+        "pygame": "pygame",
+        "pyscroll": "pyscroll",
+        "pytmx": "pytmx",
+        "requests": "requests",
+        "natsort": "natsort",
+        "PyYaml": "yaml"
+    }
+    for i in lib_entries:
+        try:
+            logging.info(f"{i}\t{__import__(lib_entries[i]).__version__}")
+        except:
+            logging.info(f"{i}\tunknown")
+
+    try:
+        logging.info(f"pygame_sdl_version\t{__import__('pygame').get_sdl_version()}")
+    finally:
+        pass
+
+    # Get file
```

Comment:
please avoid including commented code, especially if there is no note about what it is.